### ISSUE TITLE
Date format of calendar picker should follow global formatting setting

### ISSUE THREAD

**Issue by gvisniuc** [metabase/metabase#9151]
Hi there,

It would be great if we could change the datetime format for the datepicker from the admin settings.

![alt text](https://i.imgur.com/Rmzewe3.png)

The issues is with the MM-DD-YYYY format can sometimes be confusing.

![alt text](https://i.imgur.com/eAF3tba.png)

Cheers!

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by tlrobinson** [comment]
We have the "D/M/YYYY" and "YYYY/M/D" formats under the "Date style" setting, but you are right that's not currently reflected in the date picker.

![Uploading Screenshot 2018-12-19 10.15.19.png…]()

I would rather not have to add a separate "Date format" setting, but I guess there's no way to keep the date format with month names while also indicating you want to change the picker to one of the other formats. Hmm.

**Comment by gvisniuc** [comment]
Hi @tlrobinson, thanks for the response. Indeed those changes are not reflected in the date picker. 

It would be great if the date format was an option of the date picker widget. In that way it would be an isolated setting.

**Comment by flamber** [comment]
Hi @tlrobinson 

Wouldn't it be more correct to set format of the date picker to whatever the user language/country is using? Date formatting in general should be user dependent, since it doesn't make a lot of sense to set it differently for each question.

I remember Windows had the International settings in the Control Panel, which was different from Language. I used to set language to English, but set the internationalization to Denmark, so date and number formatting followed where I lived.
It's the same way Ubuntu works - I use English, but set the Region to Denmark.

I know that would be a big change from how it works today in Metabase, but currently everything feels very US-centric - hence the date picker, but also date formatting only gives 1 option to include the weekday, so I cannot have weekday-day-month-year.
